frontend/src/vault/add/AddVaultPage.tsx

```diff
@@ -0,0 +1,32 @@
+import { useTranslation } from 'react-i18next';
+import { Button } from '../../lib/ui/buttons/Button';
+import { VStack } from '../../lib/ui/layout/Stack';
+import { PageContent } from '../../ui/page/PageContent';
+import { PageHeader } from '../../ui/page/PageHeader';
+import { PageHeaderBackButton } from '../../ui/page/PageHeaderBackButton';
+import { Link } from 'react-router-dom';
+import { importVaultPath, setupVaultPath } from '../../navigation';
+import { ProductLogoBlock } from '../../ui/logo/ProductLogoBlock';
+
+export const AddVaultPage = () => {
+  const { t } = useTranslation();
+
+  return (
+    <VStack fill>
+      <PageHeader primaryControls={<PageHeaderBackButton />} />
+      <PageContent gap={16}>
+        <ProductLogoBlock />
+        <VStack gap={16}>
+          <Link to={setupVaultPath}>
+            <Button as="div">{t('create_new_vault')}</Button>
```

Comment:
what's the reasoning behind setting a button as a div? if clickable, it gotta be a button

Reply:
In HTML it's not allowed to have a `<button>` inside of `<a>`, that's why we render this button as a `div`